src/PSR7/OperationAddress.php

```diff
@@ -4,24 +4,103 @@
 
 namespace OpenAPIValidation\PSR7;
 
-class OperationAddress extends PathAddress
```

Comment:
I did not find the place where `PathAddress` was used on its own, only as `OperationAddress` or below. These changes made usage a bit simpler, since in most cases static methods were called on `OperationAddress` instance parts, so it was easier to call these methods on instance itself